lib/jsonapi_compliable/adapters/abstract.rb

```diff
@@ -211,30 +211,13 @@ def resolve(scope)
         scope
       end
 
-      # Assign these two objects together.
-      #
-      # @example Basic accessor
-      #   def associate(parent, child, association_name, association_type)
-      #     if association_type == :has_many
-      #       parent.send(association_name).push(child)
-      #     else
-      #       child.send(:"#{association_name}=", parent)
-      #     end
-      #   end
-      #
-      # +association_name+ and +association_type+ come from your sideload
-      # configuration:
-      #
-      #   allow_sideload :the_name, type: the_type do
-      #     # ... code.
-      #   end
-      #
-      # @param parent The parent object (via the JSONAPI 'relationships' graph)
-      # @param child The child object (via the JSONAPI 'relationships' graph)
-      # @param association_name The 'relationships' key we are processing
-      # @param association_type The Sideload type (see Sideload#type). Usually :has_many/:belongs_to/etc
+      # Probably want to override
       def associate(parent, child, association_name, association_type)
```

Comment:
We now default this for POROs (and likely most clients). ActiveRecord overrides it because we have to overcome AR magic and accidental queries that would fire.